Patterns/Multi-stream/Multi-stream.cpp

```diff
@@ -0,0 +1,260 @@
+/*
+Copyright (c) 2019-2020 Advanced Micro Devices, Inc. All rights reserved.
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.  IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+*/
+
+#include "helpers.hpp"
+#include <hip/hip_runtime.h>
+#include <math.h>
+#include <omp.h>
+#include <rocblas.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <vector>
+
+const int NUM_STREAMS = 4;
```

Comment:
Any reason we aren't passing this through the options?

Reply:
Thanks for the comment Daine. I referred to the sample Multi-stream example in rocBLAS-Internal where they explicitly #defined the 'NUM_THREADS' that is to be used in the program and I followed the same.

Reply:
One more reason, since i defined the NUM_STREAMS as constant i was not able to assign values through options.